src/lib/main.js

```diff
@@ -7,9 +7,44 @@ import I18n from '../i18n';
 import store from '../store';
 import constants from './constants';
 
+export const clearBusinessUnit = async () => {
+	const {
+		token,
+		config: existingConfig,
+	} = store.state;
+
+	Livechat.credentials.token = token;
+
+	const { departments } = await Livechat.config({
+		token,
+	});
+
+	await store.setState({ config: { ...existingConfig, departments }, businessUnit: undefined });
+};
+
+export const setBusinessUnit = async (newBusinessUnit) => {
+	const {
+		token,
+		config: existingConfig,
+	} = store.state;
+	if (!token) {
+		throw new Error('Error! no livechat token found. please make sure you initialize widget first before setting business unit');
+	}
+
+	Livechat.credentials.token = token;
+
+	const { departments } = await Livechat.config({
+		token,
+		businessUnit: newBusinessUnit,
+	});
+
+	await store.setState({ config: { ...existingConfig, departments }, businessUnit: newBusinessUnit });
```

Comment:
`setBusinessUnit` and `clearBusinessUnit` are almost equal, the only difference is that the first one will pass a parameter. Why don't we create a helper method that expects an optional parameter e call it within both methods? 

This way we can reduce complexity.
```suggestion
	await store.setState({ config: { ...existingConfig, departments }, ...newBusinessUnit && { businessUnit: newBusinessUnit } });
```

Reply:
Good idea!! Made this change, please check